go.mod

```diff
@@ -0,0 +1,19 @@
+module github.com/Azure-Samples/blob-storage-devguide-go
+
+go 1.19
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.2 // indirect
```

Comment:
Could we use the latest available versions (as of date) of these dependencies? Noting down a few of them
azcore v1.11.1
azidentity v1.5.2
azblob v1.3.2

Reply:
Thanks for catching this - I'll update.